apps/console/src/features/connections/pages/connections.tsx

```diff
@@ -222,7 +223,8 @@ const ConnectionsPage: FC<ConnectionsPropsInterface> = (props: ConnectionsPropsI
 
             // Hide the SMS OTP authenticator for sub organizations.
             if (authenticator.id === AuthenticatorManagementConstants.SMS_OTP_AUTHENTICATOR_ID &&
-                organizationType === OrganizationType.SUBORGANIZATION) {
+                organizationType === OrganizationType.SUBORGANIZATION &&
+                extensionsAuthenticatorConfig.disableSMSOTPInSubOrgs) {
```

Comment:
Plz check and update the other places as well

```suggestion
                extensionsAuthenticatorConfig?.disableSMSOTPInSubOrgs) {
```

Reply:
Changed